config/jobs/kubernetes-incubator/service-catalog/service-catalog-presubmits.yaml

```diff
@@ -18,3 +18,28 @@ presubmits:
         env:
         - name: NO_DOCKER
           value: "1"
+  - name: pull-service-catalog-e2e
+    agent: kubernetes
+    context: pull-service-catalog-e2e
+    always_run: true
+    skip_report: true
+    rerun_command: "/test pull-service-catalog-e2e"
+    trigger: "(?m)^/test (all|pull-service-catalog-e2e)\\s*"
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180716-9145034c9-master
+        command:
+        - make
+        args:
+        - --repo=github.com/kubernetes-incubator/service-catalog
+        - --timeout=90
+        - --scenario=execute
+        - --
+        - make
+        - test-e2e
+        env:
+        - name: KUBECONFIG
+          value: "~/.kube/config"
+        - name: SERVICECATALOGCONFIG
+          value: "~/.kube/config"
```

Comment:
I do not have any actual clue if this file exists. Where is the kubeconfig for the test to use?

Reply:
IMO inside your make rule you should do something to connect to your cluster?

Reply:
Like what? The e2e binary expects to have a kubeconfig to connect to the cluster.

Reply:
I mean, you want to connect your own test cluster, rather than using prow cluster for testing, right?

If so, you can probably do
1). if it's a gke cluster, just whitelist prow service account to your cluster
2). otherwise, you can send us your kubeconfig, we can mount it as a k8s secret
3). or you can run your own build cluster, and let prow manages your cluster and schedule your job on your cluster instead, then you can manage secret on your own cluster.

Reply:
no, I want to get prow to do everything as much as possible.

Reply:
hummm, what state of the cluster the job will try to alter? I don't think this is generally a good pattern cc @BenTheElder @cjwagner

Reply:
going to install our aggregated api server and controller into the cluster and then do some tests against it to make sure they work and reconcile appropriately.